src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -997,7 +997,7 @@
     <value>The Range-based indexer on string values produces a copy of requested portion of the string. This copy is usually unnecessary when it is implicitly used as a ReadOnlySpan or ReadOnlyMemory value. Use the AsSpan method to avoid the unnecessary copy.</value>
   </data>
   <data name="UseAsSpanInsteadOfRangeIndexerMessage" xml:space="preserve">
-    <value>Use '{0}' instead of the '{1}'-based indexer on '{2}' to avoid creating unnecessary data copies.</value>
+    <value>Use '{0}' instead of the '{1}'-based indexer on '{2}' to avoid creating unnecessary data copiesUse '{0}' instead of the '{1}'-based indexer on '{2}' to avoid creating unnecessary data copiesUse '{0}' instead of the '{1}'-based indexer on '{2}' to avoid creating unnecessary data copies</value>
```

Comment:
I will fix this duplication.

Reply:
Can you also file an issue to fix the code fix?

Reply:
I think the fix would be to change the below to use a `HashSet` instead of a `List` and make `FixInfo` implement `IEquatable<FixInfo`: https://github.com/dotnet/roslyn-analyzers/blob/be84346550158fb6933154ffaae6ae2a2e93c198/src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/Fixers/DefineDiagnosticDescriptorArgumentsCorrectlyFix.CustomFixAllProvider.cs#L97

Reply:
https://github.com/dotnet/roslyn-analyzers/pull/3923